libs/accounts/src/lib/transfer-container.tsx

```diff
@@ -6,7 +6,7 @@ import {
   useNetworkParam,
 } from '@vegaprotocol/network-parameters';
 import { useDataProvider } from '@vegaprotocol/data-provider';
-import type { Transfer } from '@vegaprotocol/wallet';
+import type { Transfer } from '@vegaprotocol/protos/dist/vega/commands/v1/Transfer';
```

Comment:
I'd prefer these types exported from the wallet. I've left a comment on vega-connectors.ts, if we make that change can we then import this type like (same goes for other files in the PR):

`import type { TransferSubmission } from '@vegaprotocol/wallet'`